src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -6349,6 +6351,122 @@ components:
         exclusive:
           $ref: '#/components/schemas/ShortIdProperty'
 
+    ##############################################################################
+    ## SNAPSHOT BUILDER API STANDARD MODELS
+    ##############################################################################
+    SnapshotBuilderConfig:
+      type: object
+      description: >
+        The configuration describing how to select criteria for issuing a snapshot builder request
+      properties:
+        domainOptions:
+          type: array
+          items:
+            $ref: '#/components/schemas/SnapshotBuilderDomainOption'
+        programDataOptions:
+          type: array
+          items:
+            $ref: '#/components/schemas/SnapshotBuilderProgramDataOption'
+        featureValueGroups:
+          type: array
+          items:
+            $ref: '#/components/schemas/SnapshotBuilderFeatureValueGroup'
+        prepackagedConceptSets:
+          type: array
+          items:
+            $ref:
+              '#/components/schemas/SnapshotBuilderPrepackagedConceptSets'
+
+    SnapshotBuilderPrepackagedConceptSets:
```

Comment:
Aren't all concept sets in the reply prepackaged? This term seems redundant to me.

Reply:
My thinking is that, for object that this is a sub-object of, there aren't any other concept sets, so calling it `Prepackaged` doesn't help to distinguish it from other concept sets. There may be other API objects that also have concept sets, but I think the `SnapshotBuilder` prefix would be enough to disambiguate it.

If this is the prepackaged concept sets for a dataset then maybe it should be `DatasetConceptSets` instead? If we have concept sets in a request in the future then it could be something like `SnapshotRequestConceptSets`

Reply:
I am fine with that naming. If we wanted to change it from prepackaged, then I would rename the variable name, rather than the type. I think the type would be correct in the current approach, just the variable name could be different.